pkg/apis/kops/validation/aws.go

```diff
@@ -196,3 +199,49 @@ func awsValidateSSLPolicy(fieldPath *field.Path, spec *kops.LoadBalancerAccessSp
 
 	return allErrs
 }
+
+func awsValidateLoadBalancerSubnets(fieldPath *field.Path, spec kops.ClusterSpec) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	lbSpec := spec.API.LoadBalancer
+
+	for i, subnet := range lbSpec.Subnets {
+		var clusterSubnet *kops.ClusterSubnetSpec
+		if subnet.Name == "" {
+			allErrs = append(allErrs, field.Invalid(fieldPath.Index(i).Child("name"), subnet, "subnet name can't be empty"))
+		} else {
+			for _, cs := range spec.Subnets {
+				if subnet.Name == cs.Name {
+					clusterSubnet = &cs
+					break
+				}
+			}
+			if clusterSubnet == nil {
+				allErrs = append(allErrs, field.Invalid(fieldPath.Index(i).Child("name"), subnet, fmt.Sprintf("subnet %q not found in cluster subnets", subnet.Name)))
+			}
+		}
+
+		if subnet.PrivateIPv4Address != nil {
+			if *subnet.PrivateIPv4Address == "" {
+				allErrs = append(allErrs, field.Invalid(fieldPath.Index(i).Child("privateIPv4Address"), subnet, "privateIPv4Address can't be empty"))
```

Comment:
```suggestion
				allErrs = append(allErrs, field.Required(fieldPath.Index(i).Child("privateIPv4Address"), subnet, "privateIPv4Address can't be empty"))
```

https://pkg.go.dev/k8s.io/apimachinery/pkg/util/validation/field#Required